sdk/containerregistry/azure-containers-containerregistry/swagger/autorest.md

```diff
@@ -96,6 +96,15 @@ directive:
     $["x-ms-enum"].modelAsString = true;
 ```
 
+### Remove "Authentication_GetAcrAccessTokenFromLogin" operation as the service team discourage using username/password to authenticate.
```

Comment:
Did we expose these methods publicly? Is this something that can be updated directly in the swagger and consider deprecating the service APIs?

Reply:
Created issue to track it https://github.com/Azure/azure-rest-api-specs/issues/23335 and will share with the service team